classes/class-log.php

```diff
@@ -110,12 +110,10 @@ public function log( $connector, $message, $args, $object_id, $context, $action,
 		}
 
 		// Prevent any meta with null values from being logged.
-		$stream_meta = array_filter(
-			$args,
-			function ( $var ) {
-				return ! is_null( $var );
-			}
-		);
+		//
+		// @see https://www.php.net/manual/en/function.array-filter.php#111091
+		$stream_meta = array_filter( $args, 'strlen' );
+		$user_meta = array_filter( $user_meta, 'strlen' );
```

Comment:
@lkraav Were you able to confirm that this works with the nested `$user_meta` array?